app/forms/onboarding/request_form.rb

```diff
@@ -29,7 +31,10 @@ def api_base_url
   end
 
   def proxy_base_url
-    URI model.sandbox_endpoint.to_s
+    path = model.backend_api_configs.first&.path || ''
```

Comment:
not sure if it should be like this:
```suggestion
    path = model.backend_api_configs.first&.path || SLASH
```

but @duduribeiro must say :smile:

Reply:
It can be both.. if you send the empty slash, it will save the `/`. But if we want to have this clear in the UI, I think `path = model.backend_api_configs.first&.path || SLASH` is better!

Reply:
According to my experiments, the `''` value is equivalent to doing nothing